examples/window.rs

```diff
@@ -1041,6 +1099,7 @@ const KEY_BINDINGS: &[Binding<&'static str>] = &[
     Binding::new("R", ModifiersState::CONTROL, Action::ToggleResizable),
     Binding::new("R", ModifiersState::ALT, Action::RequestResize),
     // M.
+    Binding::new("M", ModifiersState::empty(), Action::DumpMonitors),
```

Comment:
We generally don't bind with `::empty` because you won't be able to do regular typing. Though, it seems like a few were missed and we have `::empty` with `SetTheme` probably should change that as well to use `ALT`.

Reply:
Done, also changed the message binding.